src/ILCompiler/ilrepro/ilrepro.ilproj

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{136993D1-B753-4487-9E24-E834774B942A}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>
```

Comment:
Could you delete useless properties? I don't think RootNamespace, AllowUnsafeBlocks, StartupObject, etc. means anything to `IL.targets`.